README.md

```diff
@@ -61,10 +61,10 @@ Now that the SDK is installed, it's time to begin working with QISKit.
 
 We are ready to try out some QASM examples, which runs via the local simulator.
 
-This is a simple superposition example.
+This is a simple bell state example.
 
 ```python
-from qiskit import QuantumProgram, QISKitError, RegisterSizeError
+from qiskit import QuantumProgram, QISKitError
```

Comment:
Thanks, @stevenheidel ! However, can you actually revert the changes related to `RegisterSizeError`, and reintroduce the import and the last `except` block? While it is not an Exception from the SDK, [we actually expose it](https://github.com/stevenheidel/qiskit-sdk-py/blob/748d6af86613c020ea4c678d9aa1e76121775ccf/qiskit/__init__.py#L1) from the main import, and it encourages the practice of catching that exception that might be raised in other examples as well.